src/test/ui/parser/issues/issue-60075.stderr

```diff
@@ -11,7 +11,10 @@ LL | trait T {
    |         - item list starts here
 ...
 LL |         });
-   |           ^ non-item starts here
+   |           ^
+   |           |
+   |           non-item starts here
+   |           help: consider removing this semicolon
```

Comment:
In this case I think rustc should not emit this suggestion. The following comment I wrote explains that.

https://github.com/rust-lang/rust/blob/d43674e2c9b1a7b134ef76378c30e822c75c9aa1/compiler/rustc_parse/src/parser/item.rs#L711-L726